components/create_post/create_post.tsx

```diff
@@ -738,12 +739,6 @@ class CreatePost extends React.PureComponent<Props, State> {
             return;
         }
 
-        if (!isDirectOrGroup && trimRight(this.state.message) === '/rename') {
-            GlobalActions.showChannelNameUpdateModal(updateChannel);
```

Comment:
Does this need to be handled somewhere else like by `executeCommand`?

Reply:
actually i am not sure but we don't use Channel Name Update modal anymore.  cc @agarciamontoro

Reply:
Not sure why I was mentioned here. Maybe I'm missing some context?

Reply:
Zubair and I discussed this earlier, and it turns out that `/rename` hasn't opened that modal since at least 2017, so clearly this didn't work before